### ISSUE TITLE
installing missing binaries

### ISSUE THREAD

**Issue by nkh** [joouha/euporie#80]
only the installation with $ pipx install git+https://github.com/joouha/euporie.git@dev 

installed the binaries, the two other didn't; euphorie libs and dependencies were installed though; I've had no problems installing other applications.

**Comment by joouha** [comment]
Hi,

Which binaries are missing? Are you referring to the `euporie-notebook`, `euporie-preview`, `euporie-console` scripts?

What was the alternative installation method you tried?

Thanks

**Comment by nkh** [comment]
Don't laugh ...

I may, hrrrm, have used the wrong package name name because I thought it was "euphorie" (excuse the French ;) ) 
I just checked my history and I feel silly.

but I also tried to list the installed binaries with pip list, and did a search with grep; verified PATH, python path, ... while still not having installed the right thing so it's not a surprising result!

In the end I copied the command so it was right.

Euprorie makes a lot of sense for a Jupyter interface ! and I learned something new.
  
I any case, great work.

python kernel not working yet although installed as directed "pip3 install ipykernel" but that's another problem (and I want to use another kernel anyway)
